aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -106,7 +109,7 @@ public static String testMultiRegionAccessPointAlias() {
     return System.getenv("AWS_TEST_MULTI_REGION_ACCESS_POINT_ALIAS");
   }
 
-  public static void cleanS3Bucket(S3Client s3, String bucketName, String prefix) {
+  public static void cleanS3GeneralBucket(S3Client s3, String bucketName, String prefix) {
```

Comment:
Looks like this is called "general purpose bucket":

https://aws.amazon.com/s3/faqs/

> Q: What is an S3 general purpose bucket?
> A bucket is a container for objects stored in Amazon S3, and you can store any number of objects in a bucket. General purpose buckets are the original S3 bucket type, and a single general purpose bucket can contain objects stored across all storage classes except S3 Express One Zone. They are recommended for most use cases and access patterns.

Could you update the naming to be consistent?